pkg/config/config.go

```diff
@@ -53,9 +55,39 @@ type Config struct {
 	Pager        string   `json:"pager,omitempty" doc:"Pager command, for example 'less'. If empty no pager will be used."`
 }
 
-// Load loads the configuration from the configuration file. If the configuration file doesn't exist
-// it will return an empty configuration object.
+// Loads the configuration from the OS keyring first if available, load from the configuration file if not
 func Load() (cfg *Config, err error) {
+	if keyring, ok := IsKeyringManaged(); ok {
+		return loadFromOS(keyring)
+	}
+
+	return loadFromFile()
+}
+
+// Loads the configuration from the OS keyring. If the configuration file doesn't exist
+// it will return an empty configuration object.
```

Comment:
`configuration file` doesn't apply to this. Should we remove it from the comment?

Reply:
removed 'file' from the verbiage here